torch/_numpy/_normalizations.py

```diff
@@ -43,35 +43,35 @@
     NotImplementedType = typing.TypeVar("NotImplementedType")
 
 
-def normalize_array_like(x, parm=None):
+def normalize_array_like(x, param=None):
```

Comment:
Let's not change these numpy tests.

Reply:
These functions are not public and are not referenced outside of this file. Also, all `parm` argument is passed as positional arguments, not keywords.